apps/volunteers/views.py

```diff
@@ -282,3 +284,13 @@ def ajax_update_schedule(request):
         data[sch.section.section_id] = sch.section.name
 
     return JsonResponse(data)
+
+
+# VolunteerListView using Generic views of django
+
+class VolunteerListView(generic.ListView):
+    model=Volunteer
+    
+    context_object_name="volunteer"
+    template_name = 'volunteers/all_volunteers_list.html'
```

Comment:
Change template name also to "volunteers".

Reply:
Okay sir I will change it soon.